kolibri/core/assets/src/views/TimeDuration.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 
   <KOptionalText
-    :text="seconds ? formattedTime : ''"
+    :text="seconds !== null ? formattedTime : ''"
```

Comment:
Made this change as earlier passing `0` as `seconds` led to this expression being evaluated as `false`, and thus an empty string being displayed instead of `0 seconds`. Also added a test case related to the same.